specification/specs/common.ts

```diff
@@ -115,8 +121,11 @@ type MultiTermQueryRewrite = string
 type GeoTilePrecision = number
 type GeoHashPrecision = number
 /** Path to field or array of paths. Some API's support wildcards in the path to select multiple fields.  */
+type Field = string
 type Fields = Field | Field[]
 
+type WaitForActiveShards = byte | WaitForActiveShardOptions
```

Comment:
@Mpdreamz @swallezthis is a similar case to what we said yesterday what we should support literal types as well.
`WaitForActiveShards` should be `type WaitForActiveShards = 0 | 'all'`